### PR TITLE
run a task at a specific time of day

### PR DESCRIPTION
**Pull request by colethorsen**
-can be used in conjunction with the other frequency methods to easily set the time.

### CODE REVIEW

**Comment by MGatner** [comment]
This needs the CI\CD updates too. But also please add tests for the new method, it should be a fairly easy copy-paste from a few existing tests.

**Comment by MGatner** [comment]
@colethorsen Now that #43 is merged can you please rebase this?

File: src/FrequenciesTrait.php [hunk 1]
```diff
@@ -72,6 +72,23 @@ public function daily(?string $time = null)
         return $this;
     }
 
+    /**
+     * Runs at a specific time of the day
+     *
+     * @param string|null $time
+     *
+     * @return $this
```
Comment (MGatner) [inline src/FrequenciesTrait.php]:
```suggestion
```

**Review by MGatner — CHANGES_REQUESTED** [review]

**Comment by lonnieezell** [comment]
@MGatner is this good now?

**Comment by MGatner** [comment]
Same - probably some cleanup but let's get the content in place.

**Pull request merged**